src/main/java/com/muxiu1997/sharewhereiam/command/CommandWaypointSave.java

```diff
@@ -0,0 +1,53 @@
+package com.muxiu1997.sharewhereiam.command;
+
+import net.minecraft.command.CommandBase;
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.util.ChatComponentText;
+
+import com.muxiu1997.sharewhereiam.command.base.CommandWaypointBase;
+import com.muxiu1997.sharewhereiam.localization.Lang;
+import com.muxiu1997.sharewhereiam.mixininterfaces.IMixinWaypointStore;
+
+import journeymap.client.model.Waypoint;
+import journeymap.client.ui.UIManager;
+import journeymap.client.waypoint.WaypointStore;
+
+public class CommandWaypointSave extends CommandWaypointBase {
+
+    public static CommandWaypointSave INSTANCE = new CommandWaypointSave();
```

Comment:
I think it would be better to not store singleton instances like this and instead register with `new CommandWaypointSave()`, and same for the other commands